test/helper/helper.go

```diff
@@ -74,6 +74,29 @@ func IsDeviceStateActive(deviceId string) (bool, error) {
 	return false, err
 }
 
+func IsDeviceStateInActive(deviceId string) (bool, error) {
```

Comment:
Rather than create a separate function, the existing helper could be updated to accept a `state` as an argument; that way we can avoid creating a new function if we find a third or fourth state we need to wait for.

Reply:
updated